.github/workflows/dotnet-check.yml

```diff
@@ -24,12 +17,22 @@ jobs:
             6.0.x
       - name: Checkout files
         uses: actions/checkout@v3
+        with:
+          sparse-checkout: |
+            .github
+            dotnetv3
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v41
+        with:
+          files: "dotnetv3/**/*.cs"
```

Comment:
Changes to .csproj, .sln, or .json files could also cause the build to fail, could we include those also?

Reply:
sure thing!